### PR TITLE
[MIG][9.0] account_invoice_merge_payment module

### PR DESCRIPTION
**Pull request by cubells**
- Updated README

cc @Tecnativa

### CODE REVIEW

File: account_invoice_merge_payment/README.rst [hunk 1]
```diff
@@ -10,7 +12,28 @@ module to support fields added in Account Payment Partner from OCA/bank-payment
...
 
-* The module is automatically installed when account_invoice_merge and account_payment_partner are installed
+* The module is automatically installed when account_invoice_merge and
+account_payment_partner are installed
+
+Usage
+=====
+
+Select several invoices from the same partner and click on **Merge Partner
+Invoice** in Action button menu.
```
Comment (rafaelbn) [inline account_invoice_merge_payment/README.rst]: @cubells I've tested but I don't see this option, check:

**Comment by rafaelbn** [comment]
Please @cubells review travis and runbot in red. Thanks

File: account_invoice_merge_payment/README.rst [hunk 2]
```diff
@@ -10,7 +12,28 @@ module to support fields added in Account Payment Partner from OCA/bank-payment
 Installation
 ============
 
-* The module is automatically installed when account_invoice_merge and account_payment_partner are installed
+* The module is automatically installed when account_invoice_merge and
+account_payment_partner are installed
```
Comment (pedrobaeza) [inline account_invoice_merge_payment/README.rst]: 2 spaces before for correct indentation

File: account_invoice_merge_payment/README.rst (hunk 1 above)
Comment (pedrobaeza) [inline account_invoice_merge_payment/README.rst]: Add that the payment mode will be considered also as merging criteria (with different payment modes they won't be merged) and the merged invoice will have the payment mode filled if they are the same.

File: account_invoice_merge_payment/__openerp__.py [hunk 3]
```diff
@@ -1,40 +1,21 @@
...
     'summary': """
         Use invoice merge regarding fields on Account Payment Partner""",
-    'author': "ACSONE SA/NV,Odoo Community Association (OCA)",
+    'author': "ACSONE SA/NV, "
+              "Tecnativa, "
+              "Odoo Community Association (OCA)",
     'website': "http://acsone.eu",
     'category': 'Invoicing & Payments',
-    'version': '8.0.0.1.0',
+    'version': '9.0.0.1.0',
```
Comment (pedrobaeza) [inline account_invoice_merge_payment/__openerp__.py]: 9.0.1.0.0

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: account_invoice_merge_payment/README.rst (hunk 1 above)
Comment (cubells) [inline account_invoice_merge_payment/README.rst]: done.

File: account_invoice_merge_payment/__openerp__.py (hunk 3 above)
Comment (cubells) [reply account_invoice_merge_payment/__openerp__.py]: done.

**Review by luismontalba — APPROVED** [review]
Tested on runbot, it works fine

**Review by moylop260 — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**